src/ui/zcl_abapgit_gui_page_repo_view.clas.abap

```diff
@@ -404,11 +417,17 @@ CLASS zcl_abapgit_gui_page_repo_view IMPLEMENTATION.
 
     lo_tb_advanced = build_advanced_dropdown( iv_wp_opt = lv_wp_opt ).
 
+    lo_tb_stage = build_stage_dropdown( ).
```

Comment:
Similarly, here, "Stage" is used by everyone. It should not be a dropdown. I would put "Stage with Filter" into the "Advanced" menu.

Reply:
see above.

Reply:
I move it to the "Advanced" menu.
![image](https://user-images.githubusercontent.com/59128840/150399886-472494ef-2f56-4954-a418-d6c651b3e96a.png)

![image](https://user-images.githubusercontent.com/59128840/150399917-b054d804-270c-476f-a8eb-cafb6c9c761d.png)